symfony/scheduler/7.2/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "copy-from-recipe": {
+        "src/": "%SRC_DIR%/"
+    },
+    "aliases": ["schedule", "scheduling"]
```

Comment:
```suggestion
    "aliases": ["schedule", "scheduling"]
```
Plus "scheduler" ?

Reply:
"scheduler" already works as it's the name of the package